### DISCUSSION TITLE
Update docs to CLOVER 5 and create corresponding Wiki

### DISCUSSION THREAD

**Discussion by BenWinchester** [CLOVER-energy/CLOVER#176] [IAA]
### Issue
The CLOVER documentation needs to be updated to CLOVER 5.

The changes to CLOVER from version 4 to version 5 have significantly changed the way in which a user interacts with the software. Whilst _all_ of the internal functionality has been preserved, there have been several new APIs and user interfaces (UIs) introduced. The user guide needs to be updated to include:
* Using the new UIs,
* Using the new APIs,
* Debugging issues and understanding errors.

### Proposal
The documentation and wiki should be updated in discussion with the CLOVER development team. This will involve:
* Continuous discussion and planning around the documentation,
* The creation of a new set of corresponding wiki pages.

**Comment by phil-sandwell** [comment]
We should also align the documentation with the Wiki. I've added the first page to test it out, it seems like we can use the same documentation files for both the Wiki and Read The Docs site.

**Comment by BenWinchester** [comment]
That sounds like a good plan - I'll update the header and description to reflect this :+1: 😄 !

**Comment by BenWinchester** [comment]
Assigned @hamishbeath so this can be worked on in parallel when we all have time :smile:

**Comment by BenWinchester** [comment]
## General thoughts
As part of the IAA grant, there's scope for creating a website. It likely makes sense to raise a new issue within the IAA project grant for the creation of a website as the docs will be updated and gone through as part of that 😄 

## 🗑️ Archive 🗑️ 
This discussion will be closed/archived as
1. This issue is an ongoing issue as the docs should be continually kept up-to-date rather than updated as part of a single issue,
2. Developing a website for use within CLOVER's dissemination will cover most of the needs here.
